src/main/java/de/taimos/pipeline/aws/cloudformation/TemplateStepBase.java

```diff
@@ -182,6 +192,18 @@ public void setRollbackTriggers(String[] rollbackTriggers) {
 		return tagList;
 	}
 
+	protected final Collection<String> getAwsNotificationARNs(StepExecution stepExecution) {
```

Comment:
Where is the stepExecution used?

Reply:
> Where is the stepExecution used?

It is actually unused, I just added it for possible future feature like file with notification ARNs (similar to tags). Now it is removed.